changelog/13034.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
I wonder if the changelog file should be named for [the PR](https://github.com/hashicorp/vault/pull/12975) that updated the dep? i.e. `12975.txt`

Reply:
Ah yes, it'd be good to have that reference. Would it fail the CI check though since it won't match to this PR's number?

Reply:
Referenced in the PR, keeping it the same due to CI failures.